tests/bluetooth/common/testlib/src/adv.c

```diff
@@ -60,6 +59,16 @@ int bt_testlib_adv_conn(struct bt_conn **conn, int id, uint32_t adv_options)
 	g_ctx = &ctx;
 
 	api_err = bt_le_ext_adv_create(&param, &cb, &adv);
+	if (!api_err) {
+		struct bt_data ad;
+
+		ad.type = BT_DATA_NAME_COMPLETE;
+		ad.data_len = strlen(name);
+		ad.data = (const uint8_t *)name;
+
+		api_err = bt_le_ext_adv_set_data(adv, &ad, 1, NULL, 0);
+	}
```

Comment:
What if `name` is `NULL`? Should this be
```suggestion
	if (!api_err && name != NULL) {
		struct bt_data ad;

		ad.type = BT_DATA_NAME_COMPLETE;
		ad.data_len = strlen(name);
		ad.data = (const uint8_t *)name;

		api_err = bt_le_ext_adv_set_data(adv, &ad, 1, NULL, 0);
	}
```

Reply:
You're right, sorry I have been too quick 😅
It's updated